packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -226,6 +250,15 @@ class OpPerfTelemetry {
 					latencyStats.opPerfData.lengthInboundQueue = this.deltaManager.inbound.length;
 				}
 			}
+			if (message.contents && isRuntimeMessage(message)) {
```

Comment:
I'm not sure why you need isRuntimeMessage() check here. It's string at protocol level, even though system messages are not supposed to put content here (they will put content into `data` property, though for historical / back-compat reasons we had some summary ops putting content into both content & data properties)

Reply:
summaryAcks also send `contents`, which we should not count if we are trying to estimate collab window size. So trying to eliminate that. This check is also used by no-op heuristics logic, so continuing that pattern here.